src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Device.Gpio.Drivers
     public class LibGpiodDriver : UnixDriver
     {
         private SafeChipHandle _chip;
+		private bool _gpiochip_seted = false;
```

Comment:
Change to _gpioChipIsSet throughout update.

Reply:
also nit: indentation off by 1